ai-platform-unified/notebooks/official/gapic/ai_platform_unified_vizier_multi_objective.ipynb

```diff
@@ -0,0 +1,513 @@
+{
```

Comment:
<p>This was updated in the template so you probably didn't see it yet.</p><p> </p><p>Change to:</p><p> </p><pre class="ql-syntax" spellcheck="false">elif not os.getenv("IS_TESTING"):
    %env GOOGLE_APPLICATION_CREDENTIALS your_path_to_credentials.json
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>